.github/workflows/build-storybook.yml

```diff
@@ -55,10 +55,32 @@ jobs:
         run: aws s3 sync ./build-storybook/ s3://gdevelop-storybook/$(git rev-parse --abbrev-ref HEAD)/latest/ --delete
         working-directory: newIDE/app
 
+      - name: Compute urls
+        run: |
+          export STORYBOOK_S3_URL_COMMIT=http://gdevelop-storybook.s3-website-us-east-1.amazonaws.com/$(git rev-parse --abbrev-ref HEAD)/commit/$(git rev-parse HEAD)/index.html
+          export STORYBOOK_S3_URL_LATEST=http://gdevelop-storybook.s3-website-us-east-1.amazonaws.com/$(git rev-parse --abbrev-ref HEAD)/latest/index.html
+          echo "::set-env name=STORYBOOK_S3_URL_COMMIT::$STORYBOOK_S3_URL_COMMIT"
```

Comment:
Seems you need to use https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-environment-variable